### DISCUSSION TITLE
激活桌面端 connection refused

### DISCUSSION THREAD

**Discussion by DragonFSKY** [AlistGo/alist#7589] [Q&A]
激活时出现下面的提示 Error: Get "http://hk.nn.ci:3001/api/orderByTradeNo/xxxxxxxxxxxxxxxxxxxxxx": dial tcp 8.210.59.124:3001: connect: connection refused，用外网机器也是不行，telnet了一下貌似这个后端已经挂了
![image](https://github.com/user-attachments/assets/b61dba80-4b5a-43e6-bf77-21e9f7840813)

**Comment by xhofe** [comment]
用商家订单号，而不是支付宝的订单号。

**Comment by DragonFSKY** [comment]
可以了 多谢大佬

Andy Hsu ***@***.***> 于2024年12月2日周一 09:17写道：
